sidebars.js

```diff
@@ -14,9 +14,152 @@
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  learnSidebar: [{ type: "autogenerated", dirName: "learn" }],
-  buildSidebar: [{ type: "autogenerated", dirName: "build" }],
-  userSidebar: [{ type: "autogenerated", dirName: "user" }],
+  "learnSidebar": [
```

Comment:
all these had to be changed to ensure that the learn/learn page above (where i commented about the versioning) is not possible to get to